src/002_add_two_numbers.md

```diff
@@ -8,13 +8,12 @@ You are given two non-empty linked lists representing two non-negative integers.
 
 You may assume the two numbers do not contain any leading zero, except the number 0 itself.
 
-Example:
-
-Input: (2 -> 4 -> 3) + (5 -> 6 -> 4)
-Output: 7 -> 0 -> 8
+Example:  
+Input: (2 -> 4 -> 3) + (5 -> 6 -> 4)  
+Output: 7 -> 0 -> 8  
 Explanation: 342 + 465 = 807.
 
-## Solution in C++
+## C++ Solution 
```

Comment:
remove the space int the end of the line 16?

Reply:
It is very nice for your careful watching!